api/peripheral_api/server.js

```diff
@@ -9,9 +9,10 @@ function main() {
     __dirname + '/routes/Printer.js'
 
   ];
-  const loggingServer = new SceHttpServer(API_ENDPOINTS, 8081, '/logging_api/');
-  loggingServer.initializeEndpoints().then(() => {
-    loggingServer.openConnection();
+  const peripheralServer = new
+  SceHttpServer(API_ENDPOINTS, 8081, '/peripheral_api/');
```

Comment:
if the prefix u are doing is `peripheral_api` make sure config.example.json matches, i believe is it is `peripheralapi` atm